test/index.spec.ts

```diff
@@ -738,20 +739,45 @@ describe('onBuild()', () => {
     expect(netlifyConfig.functions['_api_*'].node_bundler).toEqual('nft')
   })
 
-  it('provides displayname for split api routes', async () => {
-    await moveNextDist()
-    await nextRuntime.onBuild(defaultArgs)
+  describe('when splitting API routes is enabled', () => {
+    let oldProcessEnv
 
-    const functionsManifest = await readJson(
-      path.join('.netlify', 'functions-internal', '___netlify-api-handler', '___netlify-api-handler.json'),
-    )
+    beforeEach(() => {
+      oldProcessEnv = process.env
+      process.env.NEXT_SPLIT_API_ROUTES = 'true'
+    })
 
-    expect(functionsManifest).toEqual({
-      config: {
-        generator: '@netlify/next-runtime@unknown',
-        name: 'Next.js API handler',
-      },
-      version: 1,
+    afterEach(() => {
+      process.env = oldProcessEnv
+    })
```

Comment:
this doesn't actually reset `process.env`, because `process.env` and `oldProcessEnv` is same object that is being mutated. 

You might want to store `process.env.NEXT_SPLIT_API_ROUTES` specifically and restore just that (tho side note if that env var would be undefined, restoring it would set `'undefined'` string because `process.env` is special thing that always coerce to string:

```
> node -e "process.env.WAT = undefined; console.log({ value: process.env.WAT });"
{ value: 'undefined' }
// ^ notice quotes around undefined
```

This won't cause problems because source code use `destr`, just quirk worth knowing about as in other situations could cause a lot of hair pulling

Reply:
Ah damn. Good point.
What about using a spread instead then?

```javascript
const oldProcessEnv = {...process.env}

// test code

process.env = oldProcessEnv
```